src/gallery/_base.scss

```diff
@@ -1,97 +1,63 @@
+@import "https://cdnjs.cloudflare.com/ajax/libs/flickity/1.2.1/flickity.min.css";
+
```

Comment:
The problem with this approach is Sass treats it as a CSS import so we take a hit with an extra HTTP request. For now, let's move a copy of the css file into our `nightshade-core` pattern folder and we can import it as we do any local css/sass file.